### ISSUE TITLE
Dead reckoning inefficient for player and monster limbs

### ISSUE THREAD

**Issue by addictgamer** [TurningWheel/Barony#300]
Right now, the code looks like this:

In game.cpp:
```
			// run entity actions
			for ( node = map.entities->first; node != nullptr; node = nextnode )

...


									// dead reckoning

...

										if ( entity->behavior == &actPlayer )
										{
											for ( node2 = map.entities->first; node2 != nullptr; node2 = node2->next )

...

										}
										if ( entity->behavior == &actMonster ) //Same thing if I am a monster.
										{

```

So it loops through the entire entity list (fine, nothing we can do about that).
Then every time it hits a player or a monster, it reloops through the entire entity list! Searching for its limbs.

It should instead grab the limbs directly from itself -- no searching!

This will improve multiplayer performance.

**Comment by lheckemann** [comment]
I've started on this for the player part (63cc1d2), which seems to work fine. This does seem to result in a bit of duplication between `std::vector<Entity*> bodyparts` and `list_t children;` but I don't know if `children` is used for anything other than body parts. My best guess is that it is, so the duplication is justified. I'll try adding this to monsters as well now.

**Comment by addictgamer** [comment]
Resolved in #305
